### PR TITLE
Coding Practice 21 

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/StreamString.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package codingPractice;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class StreamString {
+    public static void main(String[] args) {
```
Comment (anushreeSonak) [inline simple/src/codingPractice/StreamString.java:7]: please give space here

File: simple/src/codingPractice/StreamString [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Write a Java program to count the number of strings in a list that start with a specific letter using streams.
+Output:
+Original List of String of color [Red, Green, Blue, Pink, Brown]
+
+Number of colors starting with 'B': 2
+
+Number of colors starting with 'Y': 0
+
+Process finished with exit code 0
```
Comment (anushreeSonak) [inline simple/src/codingPractice/StreamString:9]: give proper name to READme file

File: simple/src/codingPractice/StreamString.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/StreamString.java:7]: done

File: simple/src/codingPractice/StreamString (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/StreamString]: Done

**Review by anushreeSonak — APPROVED** [review]

File: simple/src/codingPractice/StreamString.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+
+public class StreamString {
+
+    public static void main(String[] args) {
+
+        List<String> color = Arrays.asList("Red","Green","Blue","Pink","Brown");
+        System.out.println("Original List of String of color "+color);
+        char startingLetter = 'B';
+        long length =color.stream().filter(sort ->sort.startsWith(String.valueOf(startingLetter))).count();
```
Comment (prashantkawa) [inline simple/src/codingPractice/StreamString.java:13]: Variable name should be proper

**Review by prashantkawa — CHANGES_REQUESTED** [review]

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**